ts/components/services/ServicesTab.tsx

```diff
@@ -0,0 +1,271 @@
+    paddingHorizontal: (24 - 17) / 2 // (io-right icon width) - (io-trash icon width)
+  }
+});
+
+function renderOrganizationLogo(organizationFiscalCode: string) {
+  return (
+    <OrganizationLogo
+      logoUri={getLogoForOrganization(organizationFiscalCode)}
+      imageStyle={styles.organizationLogo}
+    />
+  );
+}
+
+function organizationContainsText(item: Organization, searchText: string) {
+  return isTextIncludedCaseInsensitive(item.name, searchText);
+}
+
+class ServicesTab extends React.PureComponent<Props> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {};
+    this.onSaveAreasOfInterest = this.onSaveAreasOfInterest.bind(this);
+    this.showChooserAreasOfInterestModal = this.showChooserAreasOfInterestModal.bind(
+      this
+    );
+    this.renderLocalQuickSectionDeletion = this.renderLocalQuickSectionDeletion.bind(
+      this
+    );
+  }
+  /**
+   * For tab Locals
+   */
+  private showChooserAreasOfInterestModal() {
+    const {
+      selectableOrganizations,
+      hideModal,
+      selectedOrganizations
+    } = this.props;
+
+    const OrganizationsList = createFactory(
+      ChooserListContainer<Organization>()
+    );
+    this.props.showModal(
+      <OrganizationsList
+        items={selectableOrganizations}
+        initialSelectedItemIds={some(new Set(selectedOrganizations || []))}
+        keyExtractor={(item: Organization) => item.fiscalCode}
+        itemTitleExtractor={(item: Organization) => item.name}
+        itemIconComponent={left(renderOrganizationLogo)}
+        onCancel={hideModal}
+        onSave={this.onSaveAreasOfInterest}
+        isRefreshEnabled={false}
+        matchingTextPredicate={organizationContainsText}
+        noSearchResultsSourceIcon={require("../../../img/services/icon-no-places.png")}
+        noSearchResultsSubtitle={I18n.t("services.areasOfInterest.searchEmpty")}
+      />
+    );
+  }
+
+  private onSaveAreasOfInterest(selectedFiscalCodes: Option<Set<string>>) {
+    if (this.props.updateOrganizationsOfInterestMetadata) {
+      if (this.props.updateToast) {
+        this.props.updateToast();
+      }
+      this.props.updateOrganizationsOfInterestMetadata(selectedFiscalCodes);
+    }
+    this.props.hideModal();
+  }
+
+  private renderLocalQuickSectionDeletion(section: ServicesSectionState) {
+    const onPressItem = () => {
+      if (this.props.userMetadata && this.props.selectedOrganizations) {
+        if (this.props.updateToast) {
+          this.props.updateToast();
+        }
+        const updatedAreasOfInterest = this.props.selectedOrganizations.filter(
+          item => item !== section.organizationFiscalCode
+        );
+        this.props.saveSelectedOrganizationItems(
+          this.props.userMetadata,
+          updatedAreasOfInterest
+        );
+      }
+    };
+    return (
+      <TouchableOpacity onPress={onPressItem}>
+        <IconFont
+          name={"io-trash"}
+          color={customVariables.brandMildGray}
+          size={17}
```

Comment:
you could define a const on top

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1409/commits/0d0048cf3f60a0289763196f8f0a087155a75d81